apps/web/src/components/settings/ShieldedAccount/index.tsx

```diff
@@ -369,57 +451,137 @@ export default function ShieldedAccount({ sx }: { sx: SxProps }) {
               <Typography variant="body2" title={mpecdhAddress}>
                 Deployed at: <code>{shortenHex(mpecdhAddress, 6)}</code>
               </Typography>
-            )}
+            )} */}
             {deploymentHash && (
               <Typography variant="body2" title={deploymentHash}>
                 Deployment tx: <code>{shortenHex(deploymentHash, 6)}</code>
               </Typography>
             )}
-            {pendingTxs.length > 0 &&
+            {/* {pendingTxs.length > 0 &&
               pendingTxs.map((item) => (
                 <Typography key={item.hash} variant="body2" title={item.hash}>
                   Pending: <code>{shortenHex(item.hash, 6)}</code> – {item.confirmationStatus}
                 </Typography>
-              ))}
+              ))} */}
             {blocking.length > 0 && (
               <Typography variant="body2">
                 Blocking: {blocking.map((addr: string) => shortenHex(addr, 4)).join(', ')}
               </Typography>
             )}
+            {mpecdhAddress && !isReady && currentRound !== null && totalRounds !== null && (
+            <Typography variant="caption" sx={{ fontSize: '0.75rem', color: 'text.secondary' }}>
+              Current round: {currentRound} | TotalRounds: {totalRounds}
+            </Typography>
+            )}
           </Stack>
+          <Box sx={{ 
+            p: 2, 
+            mt: 2,
+            backgroundColor: 'background.paper', 
+            borderRadius: 1,
+            border: '1px solid',
+            borderColor: 'divider'
+          }}>
+            <Typography variant="caption" sx={{ fontSize: '0.75rem', color: 'text.secondary', display: 'block' }}>
+            How to set up shielded account ?
+            </Typography>
+            <Typography variant="caption" sx={{ fontSize: '0.75rem', color: 'text.secondary', display: 'block', mt: 0.5 }}>
+              1. Deploy the secret sharing contract (MPECDH)
```

Comment:
The term secret sharing is more associated with key splitting techniques I think.

Would reformulate


```suggestion
              1. Deploy a multi-party key exchange coordination contract for your Safe (owners)
```